api.go

```diff
@@ -0,0 +1,45 @@
+package slack
+
+import (
+	"context"
+	"errors"
+	"net/url"
+)
+
+// APIResponseFull contains our API response from the api.test endpoint
+type APIResponseFull struct {
+	SlackResponse
+	Args map[string]string `json:"args,omitempty"`
```

Comment:
yuck. lets avoid this, we are not in a dynamic language. also according to the documentation this is effectively unused because the client needs to provide the parameters to be returned (if I am reading it properly). which you've provided no way for users to supply these parameters.